docs/UserGuide.adoc

```diff
@@ -367,3 +367,14 @@ e.g.`select 2`
 * *History* : `history`
 * *Undo* : `undo`
 * *Redo* : `redo`
+
+== Features to be included in V 2.0
```

Comment:
perhaps 'Features to be included in future versions' would be more suitable since it includes other versions as well